composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "orange-opensource/orange-boosted-bootstrap",
-  "description": "Orange Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library.",
+  "description": "Innovation Cup Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library.",
```

Comment:
"Orange branded" in this sentence shouldn't be removed ?

Reply:
Yes it should. There are still a loooot of modifications to do in the PR